server/src/test/java/org/apache/druid/server/coordinator/duty/KillUnusedSegmentsTest.java

```diff
@@ -87,6 +88,9 @@ public class KillUnusedSegmentsTest
   private static final String DS2 = "DS2";
   private static final String DS3 = "DS3";
 
+  private static final Period RETAIN_DURATION = Period.hours(6);
```

Comment:
Non-blocker: Maybe also add a test where retain duration is bigger so that it leads to smaller kill interval and thus is given precedence over the `maxIntervalToKill`.